### DISCUSSION TITLE
Artisan measurements below zero Celsius?

### DISCUSSION THREAD

**Discussion by foxlife12** [artisan-roaster-scope/artisan#1112] [Q&A]
I am having problem with Artisan as it is not showing me temperatures below zero Celsius. Once i connect my Center301 device to it graph is working perfectly normal, but once i try to read temperatures below zero, graph disappears. Anyone help??

**Comment by poundy** [comment]
Well, as coffee roasting is unlikely to see temperatures below zero, there's never been anyone to test that. And in fact, nobody would have thought about testing these cases and the application may never have been coded to take that into account.

First up, can you confirm whether you have the graph set to have axes below zero? And then demonstrate what you do see as you approach zero and go below that? Screenshots, video, anything you think would highlight what the behaviour is...
